src/amo/components/UserProfileEditNotifications/index.js

```diff
@@ -48,13 +48,15 @@ type CreateNotificationParams = {|
   label: React.Element<typeof LoadingText> | string | null,
   mandatory: boolean,
   name: string,
+  onChange?: Function,
```

Comment:
Sometimes instead of just `Function`, we specify the type using the function signature, so Flow knows exactly what the function should look like.

Reply:
I think that would make sense for custom functions, but here we only pass a React event handler. That being said, I could add a type like `(e: SyntheticEvent<HTMLInputElement>) => void` but I don't know if that would be super useful.